models.py

```diff
@@ -13,6 +13,7 @@ class Switch(BaseModel):
     pin: int = 0
     comment: bool = False
     variable: bool = False
+    label: Optional[str] = None
```

Comment:
Shouldn't this add a corresponding migration as well?

Reply:
switches is just a subclass of the basemodel it will be saved as json into the db, this will be fine.